tests/utils/clean-up.go

```diff
@@ -75,6 +75,7 @@ func RemoveAllTestOCProjects() error {
 	if err != nil {
 		return err
 	}
+	time.Sleep(20 * time.Second)
```

Comment:
Usually, sleep could create silent problems if for any reason the time needed by the process increases. But if there is no other way